ci-operator/step-registry/openshift-extended/test/openshift-extended-test-commands.sh

```diff
@@ -224,6 +224,30 @@ oc wait nodes --all --for=condition=Ready=true --timeout=15m
 oc wait clusteroperators --all --for=condition=Progressing=false --timeout=15m
 oc get clusterversion version -o yaml || true
 
+# Get the KUBEADMIN_REMOVED environment variable (or set a default false if missing)
+KUBEADMIN_REMOVED="${KUBEADMIN_REMOVED:-false}"
+# Function to delete the kubeadmin secret if KUBEADMIN_REMOVED is "true"
```

Comment:
ci-operator/step-registry/openshift-extended/test/openshift-extended-test-ref.yaml already set false by default, `KUBEADMIN_REMOVED="${KUBEADMIN_REMOVED:-false}"` can be unneeded.

Reply:
Removed